src/main/java/com/redhat/devtools/lsp4ij/features/color/LSPColorProvider.java

```diff
@@ -61,7 +61,7 @@ protected void doCollect(@NotNull PsiFile psiFile,
 
         try {
             // Wait until the future while 200ms and stop the wait if there are some ProcessCanceledException.
```

Comment:
The comment no longer matches now that you've removed the timeout.

Reply:
Good catch! fixed